alvr/client_core/src/graphics/lobby.rs

```diff
@@ -169,8 +192,11 @@ impl LobbyRenderer {
         }
     }
 
-    #[allow(unused_variables)]
-    pub fn render(&self, view_inputs: [RenderViewInput; 2]) {
+    pub fn render(
+        &self,
+        view_inputs: [RenderViewInput; 2],
+        hand_poses: [(Option<Pose>, Option<[Pose; 26]>); 2],
```

Comment:
This should only really concern itself with rendering, so it should take a precomputed list of positions and the computation part should be moved into calling code